backend/doc/cli.md

```diff
@@ -59,17 +59,37 @@ optional arguments:
   --lang LANG           set user lang (ISO 639 format)
   -p PASSWORD, --password PASSWORD
                         set user password
-  --profile PROFILE     set user profile
+  --profile PROFILE     set user profile, valid values: users, trusted-users,
+                        administrators
   --timezone TIMEZONE   set user timezone
-  --send-email          Send mail to user
+  --send-email          send mail to created user about user account creation
+                        (email parameter required an email notification need
+                        to be activated)
 ```
 
-Example:
+Example creating user with email an defined password:
+
+```
+  tracimcli user create -e "john.does@example.tld" -p "superpassword" --send-email
+```
+
+Another example with username-only user (need email.required set to false) and no password setted:
 
 ```
-  tracimcli user create -e "john@john@john.john" -p "superpassword"
+  tracimcli user create -u "john" --profile "trusted-users" --lang "fr" --public-name "John Doe"
 ```
 
+:warning: if you don't provide password and don't use --send-email, no password will be created. you can use tracimcli user update to set the password
+after user creation.
```

Comment:
I'm not sure this is a good example. 
I think its better to show warning about "not create user without password" (if done you need to update user), and give one example to create user with:`username`, `password`, `--profile`, `--lang` and `--public-name`

I think examples need to be examples of good practice.